### DISCUSSION TITLE
How to get individual traces for each sample?

### DISCUSSION THREAD

**Discussion by mistry** [lnccbrown/HSSM#597] [Q&A]
Hi All,

Love the package so far as its much faster and easier to use after I switched from HDDM. One question I had was how to view individual subject traces for each parameter, rather than just the overall trace of say, drift-rate. With HDDM, I could use `plot_posteriors`, but if I say `plot_trace` in HSSM, it doesn't seem to give me access to individual subjects, but just the averages for each parameter. I'm not sure how to tie the traces back to my input data where I can then plot the trace for a specific subject.

Any help would be appreciated!

Thanks!

**Comment by digicosmos86** [comment]
Hi!

Are you using `az.plot_trace()` in ArviZ or `model.plot_trace()`?

Thanks!
